### PR TITLE
[EigenSolversApplication] Update README.md

### PR DESCRIPTION
**Pull request by loumalouomega**
Eigen download is not necessary anymore

### CODE REVIEW

File: applications/EigenSolversApplication/README.md [hunk 1]
```diff
@@ -65,17 +63,17 @@ The application provides a generalized eigensystem solver for sparse matrices. I
...
 
-3. Build Kratos
+2. Build Kratos
 
-4. Setup the `ProjectParameters.json`
+3. Setup the `ProjectParameters.json`
 
     ```json
     "linear_solver_settings": {
         "solver_type" : "sparse_lu"
```
Comment (rubenzorrilla) [inline applications/EigenSolversApplication/README.md]:
```suggestion
        "solver_type" : "EigenSolversApplication.sparse_lu"
```

Shouldn't the solver name be preceded by the app name?

**Comment by loumalouomega** [comment]
Done @rubenzorrilla

**Comment by rubenzorrilla** [comment]
I let @oberbichler to give the final approval.

**Review by oberbichler — APPROVED** [review]
Thx!

**Pull request merged**